nixos/doc/manual/configuration/luks-file-systems.xml

```diff
@@ -37,4 +37,37 @@ Enter passphrase for /dev/disk/by-uuid/3f6b0024-3a44-4fde-a43a-767b872abe5d: ***
   on an encrypted partition, it is necessary to add the following grub option:
 <programlisting><xref linkend="opt-boot.loader.grub.enableCryptodisk"/> = true;</programlisting>
  </para>
+  <section xml:id="sec-luks-file-systems-fido2">
+  <title>FIDO2</title>
+
+  <para>
+   NixOS also supports unlocking your LUKS-Encrypted file system using a FIDO2 compatible token. In the following example, we will create a new FIDO2 credential
+   and add it as a new key to our existing device <filename>/dev/sda2</filename>:
+
+   <screen>
+# fido2luks credential $HOSTNAME
```

Comment:
This could also be `"$HOSTNAME:/dev/sda2"` or `"/dev/sda2@$HOSTNAME"` or something similar.

That said, one might argue that having more than one encrypted filesystem per host is overkill and there will always be just one, so `$HOSTNAME` might be a good fit after all ...

Reply:
This is just for the documentation, as an example. When testing with Trezor, looks like we can input only about 20 characters for that name.

Reply:
I agree that's "just the documentation", but we should try to be as correct as possible there because most of the users just copy-paste from the documentation. But as I said, I am fine with just using the `$HOSTNAME`.

Reply:
You are right. I will add in an obvious way it's an example, so people are forced to change it and not be keen to copy pasting.